superset/jinja_context.py

```diff
@@ -296,7 +297,7 @@ def __init__(
             self._schema = table.schema
         self._extra_cache_keys = extra_cache_keys
         self._context: Dict[str, Any] = {}
-        self._env = SandboxedEnvironment()
+        self._env = SandboxedEnvironment(undefined=DebugUndefined)
```

Comment:
Is `DebugUndefined` required to make `find_undeclared_variables` function? Doesn't seem to be a requirement in the docs.